codefresh.yml

```diff
@@ -118,14 +118,19 @@ steps:
         title: Python build
         fail_fast: false
         image: python:3.10-alpine
-        working_directory: ./rosetta-source/target/classes/cdm/python
+        working_directory: ./rosetta-source
         shell: sh
         commands:
           - python3 -m pip install pydantic
           - python3 -m pip install jsonpickle
-          - python3 -m pip install rosetta_runtime-1.0.0-py3-none-any.whl
-          - |-
-            python3 -m pip wheel --no-deps --only-binary :all: .
+          - python3 -m pip install ./target/classes/cdm/python/runtime/rosetta_runtime-1.0.0-py3-none-any.whl
+          - python3 -m pip wheel --no-deps --only-binary :all: --wheel-dir ./target/classes/cdm/python ./target/classes/cdm/python
```

Comment:
This line has clashes with the codefresh syntax.
```
- python3 -m pip wheel --no-deps --only-binary :all: --wheel-dir ./target/classes/cdm/python ./target/classes/cdm/python
```
You can escape the clash with this:
```
          - |-
            python3 -m pip wheel --no-deps --only-binary :all: --wheel-dir ./target/classes/cdm/python ./target/classes/cdm/python
```
e.g.
```
        commands:
          - python3 -m pip install pydantic
          - python3 -m pip install jsonpickle
          - python3 -m pip install ./target/classes/cdm/python/runtime/rosetta_runtime-1.0.0-py3-none-any.whl
          - |-
            python3 -m pip wheel --no-deps --only-binary :all: --wheel-dir ./target/classes/cdm/python ./target/classes/cdm/python
          - python3 -m pip install ./target/classes/cdm/python/python_cdm-*-py3-none-any.whl
          - python3 -m pip install pytest
          - pytest ./src/test/python/
          - rm -rf ./src/test/python/serialization/__pycache__ ./src/test/python/semantics/__pycache__ 
```
Further details:
https://codefresh.io/docs/docs/pipelines/what-is-the-codefresh-yaml/#escaping-strings